client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -210,13 +211,34 @@ public abstract void mapPartitionMapperEnd(
    * @return
    * @throws IOException
    */
-  public abstract CelebornInputStream readPartition(
+  public CelebornInputStream readPartition(
       int shuffleId,
       int partitionId,
       int attemptNumber,
       int startMapIndex,
       int endMapIndex,
       MetricsCallback metricsCallback)
+      throws IOException {
+    return readPartition(
+        shuffleId,
+        partitionId,
+        attemptNumber,
+        startMapIndex,
+        endMapIndex,
+        shuffleId,
+        metricsCallback,
+        null);
```

Comment:
Incorrect parameter order ?
```suggestion
    return readPartition(
        shuffleId,
        shuffleId,
        partitionId,
        attemptNumber,
        startMapIndex,
        endMapIndex,
        metricsCallback,
        null);
```